docs/usage.md

```diff
@@ -519,15 +592,19 @@ rm *dmp *txt *gz *prt *zip
 
 </details>
 
+A detailed description can be found [here](https://github.com/bbuchfink/diamond/wiki/1.-Tutorial)
+
 #### Kaiju custom database
 
-It is possible to [create custom databases](https://github.com/bioinformatics-centre/kaiju#custom-database) with Kaiju. To build a kaiju database, you need three components: a FASTA file with the protein sequences (the headers are the numeric NCBI taxon identifiers of the protein sequences), number of threads and you need to define the uppercase characters of the standard 20 amino acids.
+To build a kaiju database, you need three components: a FASTA file with the protein sequences (the headers are the numeric NCBI taxon identifiers of the protein sequences), and you need to define the uppercase characters of the standard 20 amino acids you wish to include.
```

Comment:
If we exclude the number of threads from the text, shall we maybe write two components instead?